Exiled.API/Extensions/MirrorExtensions.cs

```diff
@@ -0,0 +1,299 @@
+
+        /// <summary>
+        /// Gets a all DirtyBit <see cref="ulong"/> from <see cref="String"/>(format:classname.methodname).
+        /// </summary>
+        public static ReadOnlyDictionary<string, ulong> SyncVarDirtyBits
+        {
+            get
+            {
+                if (SyncVarDirtyBitsValue.Count == 0)
+                {
+                    foreach (var property in typeof(ServerConsole).Assembly.GetTypes()
+                        .SelectMany(x => x.GetProperties())
+                        .Where(m => m.Name.StartsWith("Network")))
+                    {
+                        var bytecodes = property.GetSetMethod().GetMethodBody().GetILAsByteArray();
+                        if (!SyncVarDirtyBitsValue.ContainsKey($"{property.DeclaringType.Name}.{property.Name}"))
+                             SyncVarDirtyBitsValue.Add($"{property.DeclaringType.Name}.{property.Name}", bytecodes[bytecodes.LastIndexOf((byte)OpCodes.Ldc_I8.Value) + 1]);
+                    }
+                }
+
+                return ReadOnlySyncVarDirtyBitsValue;
+            }
+        }
+
+        /// <summary>
+        /// Gets a <see cref="NetworkBehaviour.SetDirtyBit(ulong)"/>'s <see cref="MethodInfo"/>.
+        /// </summary>
+        public static MethodInfo SetDirtyBitsMethodInfo
+        {
+            get
+            {
+                if (setDirtyBitsMethodInfoValue == null)
+                {
+                    setDirtyBitsMethodInfoValue = typeof(NetworkBehaviour).GetMethod(nameof(NetworkBehaviour.SetDirtyBit));
+                }
+
+                return setDirtyBitsMethodInfoValue;
+            }
+        }
+
+        /// <summary>
+        /// Gets a NetworkServer.SendSpawnMessage's <see cref="MethodInfo"/>.
+        /// </summary>
+        public static MethodInfo SendSpawnMessageMethodInfo
+        {
+            get
+            {
+                if (sendSpawnMessageMethodInfoValue == null)
+                {
+                    sendSpawnMessageMethodInfoValue = typeof(NetworkServer).GetMethod("SendSpawnMessage", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Static);
+                }
+
+                return sendSpawnMessageMethodInfoValue;
+            }
+        }
+
+        /// <summary>
+        /// Shaking target <see cref="Player"/> window.
+        /// </summary>
+        /// <param name="player">Target to shake.</param>
+        public static void Shake(this Player player) => SendFakeTargetRpc(player, AlphaWarheadController.Host.netIdentity, typeof(AlphaWarheadController), nameof(AlphaWarheadController.RpcShake), false);
+
+        /// <summary>
+        /// Play beep sound to <see cref="Player"/>.
+        /// </summary>
+        /// <param name="player">Target to play.</param>
+        public static void PlayBeepSound(this Player player) => SendFakeTargetRpc(player, ReferenceHub.HostHub.networkIdentity, typeof(AmbientSoundPlayer), nameof(AmbientSoundPlayer.RpcPlaySound), 7);
```

Comment:
Couldn't this be changed in a more generic `PlaySound`?

Reply:
Don't call it just `PlaySound`, it's confusing as it has no other parameter, but `Beep` means the sound from the intercom.

Reply:
Yes, that's why I would call it PlaySound because he should add a parameter to play more sounds than the intercom one.
I don't know if it's possible tho.